packages/orchestrator-ui-components/src/components/confirmationDialog/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
If only used inside forms, move it to the component folder of Forms

Reply:
Better do it in https://github.com/workfloworchestrator/orchestrator-ui/issues/318